README.md

```diff
@@ -65,14 +65,19 @@ deb [arch=amd64] https://downloads.apache.org/cassandra/debian 311x main
 
 Note that Redhat variants like Amazon Linux 2 avoid this issue. In our out of box Cassandra performance testing (early July 2020), Amazon Linux 2 using [the Corretto 8 JDK](https://docs.aws.amazon.com/corretto/latest/corretto-8-ug/amazon-linux-install.html) outperformed Ubuntu 20.04 by up to 23%.
 
-## Bazel on Debian-based distros
-The [Bazel build tool](https://www.bazel.build/) now releases a pre-built binary for arm64, but as of August 2020, this is not available in their custom Debian repo. Instead, we recommend using the [Bazelisk installer](https://github.com/bazelbuild/bazelisk/releases):
+## Bazel on Linux
+The [Bazel build tool](https://www.bazel.build/) now releases a pre-built binary for arm64. As of August 2020, this is not available in their custom Debian repo, and Bazel does not officially provide an RPM. Instead, we recommend using the [Bazelisk installer](https://docs.bazel.build/versions/master/install-bazelisk.html), will replace your `bazel` command and [keep bazel up to date](https://github.com/bazelbuild/bazelisk/blob/master/README.md).
+
+Below is an example using the [latest Arm binary release of Bazelisk](https://github.com/bazelbuild/bazelisk/releases) as of August 2020:
```

Comment:
try this link:  https://github.com/bazelbuild/bazelisk/releases/latest

it gets the user straight to the latest